update-execution-platform-command.js

```diff
@@ -0,0 +1,43 @@
+const EXECUTION_PLATFORM_NAME = 'modeler:executionPlatform',
```

Comment:
Maybe to consider, but not needed: We tend to have a folder structure for extensions. Something like

```text
- src/
  - cmd/
    - UpdateExecutionPlatformHandler.js
    - index.js
  - ExecutionPlatform.js
  - index.js  
```

Maybe for this extension it's okay to keep things in the root.

Reply:
I can do that.

Reply:
OK now it's the canonical way: https://github.com/bpmn-io/execution-platform/pull/3/commits/166fa632aaf942b63c8fbf00d72e23365eea6bdc

Reply:
Nice! Thanks